### PR TITLE
Add global disable/enable event mirroring

### PR DESCRIPTION
**Pull request by durgakiran**
# ✨ Pull Request

### 📓 Referenced Issue

Implements #317 

### ℹ️ About the PR

Adds a new event `TOGGLE_EVENT_MIRRORING_ALL_DEVICES` which disables/enables event mirroring across the Webviews. The default assumption is event mirroring is **on**.  This event does not toggle individually, instead, it does enable/disable all the Webviews at once.

### 🖼️ Testing Scenarios / Screenshots
Disabled state:
disabled event mirroring

Enabled state:
enabled event mirroring state

### CODE REVIEW

**Comment by durgakiran** [comment]
I could not find any testing for pubsub events. Did I miss anything regards to pubsub events or intentionally left like that?

**Comment by manojVivek** [comment]
> I could not find any testing for pubsub events. Did I miss anything regards to pubsub events or intentionally left like that?

@durgakiran We didn't build any test cases for those yet, that is why.

File: desktop-app/app/components/ScrollControls/index.js [hunk 1]
```diff
@@ -55,6 +60,18 @@ const ScrollControls = ({
     width: 25,
   };
 
+  useEffect(() => {
```
Comment (manojVivek) [inline desktop-app/app/components/ScrollControls/index.js:63]: @durgakiran Why do we need this?

Comment (durgakiran) [reply desktop-app/app/components/ScrollControls/index.js:63]: To run whenever eventMirroring value changes and initial render check is to make sure toggle does not happen on  initial render

**Comment by manojVivek** [comment]
@durgakiran Thanks so much for the PR, merging this in now.

**Comment by manojVivek** [comment]
@all-contributors Please add @durgakiran for code.

**Pull request merged**